apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx

```diff
@@ -41,6 +47,20 @@ const ContributorsList = ({
 		[onMergeContributor],
 	);
 
+	const contributorActivitiesMap = useMemo(() => {
+		const activitiesMap = new Map<number, ContributorActivity | undefined>();
+
+		for (const contributor of contributors) {
+			const contributorActivity = activityLogs.find(
+				(activityLog) => Number(activityLog.contributorId) === contributor.id,
+			)?.commitsNumber;
+
+			activitiesMap.set(contributor.id, contributorActivity);
+		}
```

Comment:
we don't need the logic with matching contributors here. We can just go through activityLogs and set activityLog.contributorId as the key. activityLogs will already contain only needed contributor ids and we don't want to search through the array for each contributor if we can avoid this.

Reply:
Sure